includes/views/admin.php

```diff
@@ -1,88 +1,106 @@
 		</td>
 	</tr>
 
 	<tr>
-		<th scope="row"><p><b><?php _e( 'Footer Output', 'genesis-simple-edits' ); ?></b></p></th>
+		<th scope="row"><p><b><?php esc_html_e( 'Footer Output', 'genesis-simple-edits' ); ?></b></p></th>
 		<td>
-			<p><input type="checkbox" name="<?php $this->field_name( 'footer_output_on' ); ?>" id="<?php $this->field_id( 'footer_output_on' ); ?>" value="1" <?php checked( 1, $this->get_field_value( 'footer_output_on' ) ); ?> /> <label for="<?php $this->field_id( 'footer_output_on' ); ?>"><?php _e( 'Modify Entire Footer Text (including markup)?', 'genesis-simple-edits' ); ?></label></p>
+			<p><input type="checkbox" name="<?php $this->field_name( 'footer_output_on' ); ?>" id="<?php $this->field_id( 'footer_output_on' ); ?>" value="1" <?php checked( 1, $this->get_field_value( 'footer_output_on' ) ); ?> /> <label for="<?php $this->field_id( 'footer_output_on' ); ?>"><?php esc_html_e( 'Modify Entire Footer Text (including markup)?', 'genesis-simple-edits' ); ?></label></p>
 
-			<p><span class="description"><?php _e( 'NOTE: Checking this option will use the content of the box below, and override the options above.', 'genesis-simple-edits' ); ?></span></p>
+			<p><span class="description"><?php esc_html_e( 'NOTE: Checking this option will use the content of the box below, and override the options above.', 'genesis-simple-edits' ); ?></span></p>
 
 			<p><textarea name="<?php $this->field_name( 'footer_output' ); ?>" cols="80" rows="5"><?php echo esc_textarea( $this->get_field_value( 'footer_output' ) ); ?></textarea></p>
 
-			<p><small><a class="footer-shortcodes-toggle" href="#"><?php _e( 'Show available footer shortcodes', 'genesis-simple-edits' ); ?></a></small></p>
+			<p><small><a class="footer-shortcodes-toggle" href="#"><?php esc_html_e( 'Show available footer shortcodes', 'genesis-simple-edits' ); ?></a></small></p>
 		</td>
 	</tr>
 
 	<tr class="footer-shortcodes" style="display: none;">
-		<th scope="row"><p><span class="description"><?php _e( 'Shortcode Reference', 'genesis-simple-edits' ); ?></span></p></th>
+		<th scope="row"><p><span class="description"><?php esc_html_e( 'Shortcode Reference', 'genesis-simple-edits' ); ?></span></p></th>
 		<td>
-			<p><span class="description"><?php printf( __( 'NOTE: For a more comprehensive shortcode usage guide, see the <a href="%s" target="_blank">footer shortcode reference</a>.', 'genesis-simple-edits' ), 'http://my.studiopress.com/documentation/customization/shortcodes-reference/footer-shortcode-reference/' ); ?></span></p>
+			<p><span class="description"><?php // Translators: The string is a URL to a usage guide. ?><?php printf( esc_attr( __( 'NOTE: For a more comprehensive shortcode usage guide, see the <a href="%s" target="_blank">footer shortcode reference</a>.', 'genesis-simple-edits' ) ), 'http://my.studiopress.com/documentation/customization/shortcodes-reference/footer-shortcode-reference/' ); ?></span></p>
```

Comment:
Here, we're using `esc_attr` but on line 40+, there's different formatting and `wp_kses` is used.